platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -1187,27 +1195,87 @@ bool OfflineDatabase::evict(uint64_t neededFreeSize) {
         tileQuery.run();
         const uint64_t tileChanges = tileQuery.changes();
 
+        // Update current ambient cache size, based on diff from original size.
+        const uint64_t releasedBytes = initialDatabaseSize - currentDatabaseSize();
+        newAmbientCacheSize = releasedBytes > newAmbientCacheSize ? 0 : newAmbientCacheSize - releasedBytes;
```

Comment:
s/newAmbientCacheSize/nextmbientCacheSize ?

Reply:
(releasedBytes > newAmbientCacheSize)

Reply:
`>=` ?

Reply:
sure, result is the same, but >= is cleaner.